File: hw/villagomez/u1/hw12CalculateIR/IncomeIr.js/ec.edu.espe.incomeir.view/IncomeIR.js
const readline = require('readline');

const { computeIR } = require('../../Taxes/taxes/Income.js');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

function main() {
    console.log("Calculate of Impuesto a la Renta");
  
    rl.question("Enter your monthly salary: ", (salary) => {
      const salaryNumber = parseFloat(salary);
  
      const taxPerYear = computeIR(salaryNumber);
  
      console.log("The annual IR is: " + taxPerYear);
      console.log("The monthly IR is: " + taxPerYear / 12);
      console.log("The monthly salary with IR is: " + (salaryNumber - (taxPerYear / 12)));
      console.log("The annual salary with IR is: " + ((salaryNumber * 12) - taxPerYear));
  
      rl.close();
    });
  }
  
main();

File: hw/villagomez/u1/hw12CalculateIR/hw12IncomeIR/src/ec/edu/espe/incomeir/view/IncomeIR.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.incomeir.view;

import java.util.Scanner;

/**
 *
 * @author Doménica Villagómez, CommitCrew, DCCO-ESPE
 */
public class IncomeIR {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("Calculate of Impuesto a la Renta");
        System.out.println("Enter your monthly salary: ");
        float salary = input.nextFloat();
        
        float taxPerYear = taxes.Income.computeIR(salary);
        
        System.out.println("The annual IR is: " + taxPerYear);
        System.out.println("The monthly IR is: " + taxPerYear/12);
        
        System.out.println("The monthly salary with IR is: " + (salary - (taxPerYear/12)));
        System.out.println("The annual salary with IR is: " + ((salary*12) - taxPerYear));
        
    }
}
